modules/core/src/main/kotlin/datamaintain/core/config/DatamaintainConfig.kt

```diff
@@ -64,10 +66,19 @@ data class DatamaintainConfig @JvmOverloads constructor(val path: Path = Paths.g
                     executionMode,
                     scriptAction,
                     driverConfig,
-                    props.getProperty(VERBOSE).toBoolean()
+                    props.getProperty(VERBOSE).toBoolean(),
+                    extractFlags(props.getNullableProperty(FLAGS))
             )
         }
 
+        private fun extractFlags(flags: String?): List<String> {
+            return if (flags.isNullOrEmpty()) {
+                listOf()
+            } else {
+                flags.split(",")
+            }
+        }
```

Comment:
I'm quite bothered by the many ways list options are parsed. There are three functions that do pretty much the same thing (except for ```extractTags``` that does the split and then maps it to tags)... I'm sure there is a way to remove such duplication.